custom_components/unifi_insights/api/protect/endpoints/alarm_hubs.py

```
@@ -0,0 +1,143 @@
+        """
+        List all alarm hubs.
+
+        Args:
+            site_id: The site ID (required for REMOTE connections, ignored for LOCAL).
+
+        Returns:
+            List of alarm hubs.
+
+        """
+        path = self._client.build_api_path("/alarm-hubs", site_id)
+        response = await self._client._get(path)
```

Pylint (4.0.6)

[warning] Access to a protected member _get of a client class

(W0212)